workers/generic-worker/README.md

```diff
@@ -1,782 +1,12 @@
-    apt install -y docker-ce
-    sleep 5
-    systemctl status docker | grep "Started Docker Application Container Engine"
-    usermod -aG docker ubuntu
-    ```
-
- 5. Download `generic-worker`, `taskcluster-proxy` and `livelog` to `/usr/local/bin`:
-
-    ```
-    cd /usr/local/bin
-    curl -L "https://github.com/taskcluster/generic-worker/releases/download/<GENERIC_WORKER_VERSION>/generic-worker-multiuser-linux-amd64" > generic-worker
-    curl -L "https://github.com/taskcluster/livelog/releases/download/<LIVELOG_VERSION>/livelog-linux-amd64" > livelog
-    curl -L "https://github.com/taskcluster/taskcluster-proxy/releases/download/<TASKCLUSTER_PROXY_VERSION>/taskcluster-proxy-linux-amd64" > taskcluster-proxy
-    ```
-
- 6. Make binaries executable:
-
-    ```
-    chmod a+x /usr/local/bin/{generic-worker,taskcluster-proxy,livelog}
-    ```
-
- 7. Create directories required by generic-worker:
-
-    ```
-    mkdir -p /etc/generic-worker /var/local/generic-worker
-    ```
-
- 8. Check generic-worker works and has correct version:
-
-    ```
-    /usr/local/bin/generic-worker --version
-    ```
-
- 9. Generate a key for signing artifacts:
-
-    ```
-    /usr/local/bin/generic-worker new-ed25519-keypair --file /etc/generic-worker/ed25519_key
-    ```
-
-    The private key will be written to the file
-    `/etc/generic-worker/ed25519_key`, and the public key will be written to
-    standard out. Keep a copy of the public key if you wish to validate artifact
-    signatures.
-
-10. Ensure host 'taskcluster' resolves to localhost:
-
-    ```
-    echo 127.0.1.1 taskcluster >> /etc/hosts
-    ```
-
-11. __AWS workers only__: configure generic-worker to run on boot with
-	dynamically provided config from AWS Provisioner:
-
-    ```
-    echo '@reboot cd /var/local/generic-worker && PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /usr/local/bin/generic-worker run --configure-for-aws --config /etc/generic-worker/config >> /var/log/generic-worker.log 2>&1' | crontab -
-    ```
-
-	Make sure netplan network renderer takes precendence over eni network
-	renderer. See [bug 1499054 comment
-    12](https://bugzilla.mozilla.org/show_bug.cgi?id=1499054#c12) for an
-    explanation.
-
-    ```
-    cat > /etc/cloud/cloud.cfg.d/01_network_renderer_policy.cfg << EOF
-    system_info:
-        network:
-          renderers: [ 'netplan', 'eni', 'sysconfig' ]
-    EOF
-    ```
-
-12. __Non-AWS workers only__: configure generic-worker to run on boot with
-	a static local config file:
-
-    ```
-    echo '@reboot cd /var/local/generic-worker && PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /usr/local/bin/generic-worker run --config /etc/generic-worker/config >> /var/log/generic-worker.log 2>&1' | crontab -
-    ```
-
-    Create `/etc/generic-worker/config` with appropriate configuration settings
-    (see `generic-worker --help` for details).
-
-
-# Acquire taskcluster credentials for running tests
```

Comment:
The general idea of the monorepo is that contributors should get reasonably accurate testing results from running without credentials, so that in most cases they can make a PR based only on credential-free tests and see green checkmarks.  

But, generic-worker's not there yet, so it makes sense to have some information on how to get those scopes.  However, this section was quite out-of-date, and I'm not sure what the new system is.  I'll put a short section back in, and leave it to you to set up that process.